src/index.js

```diff
@@ -279,7 +284,8 @@ const initialState = {
   isDragReject: false,
   draggedFiles: [],
   acceptedFiles: [],
-  rejectedFiles: []
+  rejectedFiles: [],
+  rejectReasons: []
```

Comment:
I think we should replace `rejectedFiles` w/ `rejectedReasons`. So the on drop cb (and others) becomes:
```ts
onDrop(files Array<File>, errors Array<FileError>, evt Event)
```

Reply:
i went with `fileRejections` over `rejectedFiles` because I think a rejection is more representative of what it is. To me, a rejected file would just be comprised of the file itself. it wouldn't include an `errors` property. I'll be happy to change it to `rejectedFiles` if you don't like the change though. Lmk :)